Solutions/Oracle Cloud Infrastructure/Data Connectors/AzureFunctionOCILogs/main.py

```diff
@@ -65,7 +65,15 @@ def determine_log_type(event):
     #     return "OCI_LoadBalancerLogs"
     if event["type"] == "com.oraclecloud.Audit.ListEvents":
         return "OCI_AuditLogs"
-    if event["type"] == "com.oraclecloud.vcn.flowlogs.DataEvent" or event["type"] == "com.oraclecloud.vcn.flowlogs.QualityEvent.NoData" or event["type"] == "com.oraclecloud.virtualNetwork.GetVcn" or event["type"] == "com.oraclecloud.virtualNetwork.ListVcns" or event["type"] == "com.oraclecloud.vcn.flowlogs.QualityEvent.SkipData" or event["type"] == "com.oraclecloud.virtualNetwork.GetVcnDnsResolverAssociation":
+    if event["type"] == "com.oraclecloud.computeApi.ListVolumeAttachments" or event["type"] == "com.oraclecloud.computeApi.ListVnicAttachments" or event["type"] == "com.oraclecloud.computeApi.ListBootVolumeAttachments" or event["type"] == "com.oraclecloud.computeApi.GetInstance":
```

Comment:
As i said before we should do a  contains in order to classifiy each log type instead of parssing each type, 

reason: 
in the future there is an option that it may be a new event data type for example :
com.oraclecloud.vcn.newqtype

because we are using == instead of **in** this log will be sent to the defualt talbe and not to the relevant one